x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx

```diff
@@ -57,18 +57,21 @@ const AlertSummaryViewComponent: React.FC<{
   eventId: string;
   isDraggable?: boolean;
   timelineId: string;
-  title?: string;
-}> = ({ browserFields, data, eventId, isDraggable, timelineId, title }) => {
+  title: string;
```

Comment:
💅  `title` was defined in all cases apart from tests. Made it not optional and updated the test data.